text/0000-template.md

```diff
@@ -0,0 +1,159 @@
+# A Dynamic Configuration Service for the SDK
+
+This proposal is for an experiment to add a configuration service to dynamically configure metric  periods. Per-metric and tracing configuration is also intended to be added, with details left for a later iteration.
```

Comment:
```suggestion
This proposal is for an experiment to add a configuration service to dynamically configure metric  periods. Per-metric and tracing configurations are also intended to be added, with details left for a later iteration.
```

Reply:
Do you think this might be ok? In that the word "configuration" is the concept, and the word "configurations" is the noun.